### PR TITLE
Wager label accuracy

### PR DESCRIPTION
**Pull request by ryanprice**
Small change to the tooltip
There may be a more elegant way to code this, suggestions welcome :)

### CODE REVIEW

File: src/components/CryptContent.vue [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
         <b-progress show-progress :max="pBarMax" height="30px" class="mb-3">
           <b-progress-bar
-            v-b-tooltip.hover="`${pBarDefaultValues[0]}% chance to pull an Epic`"
+            v-b-tooltip.hover="`0.01% chance to pull an Epic`"
```
Comment (mchigit) [inline src/components/CryptContent.vue:16]: I believe this should be `0.04` for all 4 rarities to add up to 100%

**Review by mchigit — COMMENTED** [review]
Not sure if this is intended, but when the wager amount is lower than 1000000 the probablity doesn't change?

image

Also, the 0 wager probability doesn't match what's stated above anymore, which might generate confusion for users?

image

File: src/components/CryptContent.vue (hunk 1 above)
Comment (ryanprice) [reply src/components/CryptContent.vue:16]: check changes below and see if it makes sense

**Comment by ryanprice** [comment]
> Not sure if this is intended, but when the wager amount is lower than 1000000 the probablity doesn't change?
> 
> image
> 
> Also, the 0 wager probability doesn't match what's stated above anymore, which might generate confusion for users?
> 
> image

Correct, below 1mm we default back to the base probability

**Comment by mchigit** [comment]
hmm on a side note, I don't think there should be that many changes to `yarn.lock`, since you didn't add any packages. But I don't think that should be a huge issue

**Review by mchigit — APPROVED** [review]

**Pull request merged**